### DISCUSSION TITLE
Trying to add filament Tables to an existing laravel 8.83.27 project trigger error on loading page

### DISCUSSION THREAD

**Discussion by faridsa** [filamentphp/filament#5926] [Help]
Hello fellas, I got a Laravel project which uses Livewire in admin and Vuejs + Inertia in frontend and it is working fine.
Now I'm trying to add filament Tables to use it in admin list. 
After install as this guide describe https://filamentphp.com/docs/2.x/tables/installation#laravel-mix my frontend returns the following errors

```
Uncaught TypeError: Cannot read properties of undefined (reading 'directive')
    at 588 (app.js:2:850757) 
    at o (manifest.js:1:127)
    at e (app.js:2:896739)
    at app.js:2:896770
    at o.O (manifest.js:1:393)
    at app.js:2:896789
    at e (manifest.js:1:969)
    at app.js:2:56
```

Following sources links in errors I can see that aparently this is the line that triggers the error 
`window.Livewire.directive("sortable", (function(t) {`

Any ideas on how to solve this?

**Comment by danharrin** [comment]
To use the table builder inside the admin panel, it doesn't need to be installed. It's already installed for you there. You only need to install it if you want to use it on the frontend.

**Reply by faridsa** [reply]
Thank you for your assistance Dan. My admin panel was made using livewire before knowing about the existence of Filament, so it is made using livewire and blade files in an old way. After meet Filament I read that table builder could be installed as a standalone package, and that was what I tried.

**Reply by danharrin** [reply]
So you're not using the Filament admin panel, it's a custom one? Have you fixed your issue?

**Reply by faridsa** [reply]
Not yet, I've realized that using filament tables I will need separate laravel mix tasks for frontend and backend because I using livewire for backend and vue+inertia for the frontend views, so I'm now looking for the way to achieve that. I have read that minimist could help so I will investigate it.
Thank you again for your kindly support! I will further report later
